include/glow/Backends/BackendUtils.h

```diff
@@ -59,6 +65,8 @@ class RuntimeBundle {
   size_t getValueOffset(const Named *v) const;
   /// Helper function, gets symbol info for \p v.
   RuntimeSymbolInfo getSymbolInfo(const Named *v) const;
+  /// Get a const reference to the symbol table.
```

Comment:
Channeling @bertmaher and since I'm a frequent transgressor, remember vertical whitespace between declarations.